tests/test_type_inference/test_subscript.py

```diff
@@ -41,6 +42,31 @@ def test_subscript_homogeneous_list_slice(node):
 #             assert subscript_node.inf_type.getValue() == list(dict_node.items)[0][1].inf_type. getValue()
 
 
+@given(cs.simple_homogeneous_list_node(min_size=1))
+def test_inference_invalid_slice(node):
+    sub_node = astroid.Subscript()
+    slice = astroid.Slice()
+    slice.postinit(astroid.Const(0), astroid.Const('a'))
+    sub_node.postinit(node, slice)
+    module, _ = cs._parse_text(sub_node)
+    for subscript_node in module.nodes_of_class(astroid.Subscript):
+        assert isinstance(subscript_node.inf_type, TypeFail)
+
+
+@given(cs.simple_homogeneous_list_node(min_size=1))
+def test_inference_invalid_ext_slice(node):
```

Comment:
Remove this test. If you want to test the `ExtSlice` node here, give a valid one (e.g., `x[1, 2]`) and check that it has `type_inf` `NoType`.